docs/source/building/meta-yaml.rst

```diff
@@ -525,6 +525,243 @@ The script run_test.sh (or .bat/.py/.pl) will be run automatically if it is part
 
 NOTE: Python or Perl .py/.pl scripts are only valid as part of Python/Perl packages, respectively.
 
+
+Outputs section
+---------------
+
+Explicitly specify packaging steps. Supports multiple outputs, as well as
+different package output type. Format is a list of mappings. Build strings for
+subpackages are determined by their runtime dependencies. This support was added
+in conda-build 2.1.0.
+
+.. code-block:: yaml
+
+   outputs:
+     - name: some-subpackage
+     - name: some-other-subpackage
+
+
+NOTE: if any output is specified in the outputs section, the default packaging
+behavior of conda-build is bypassed. In other words, if any subpackage is
+specified, then you will not get the normal top-level build for this recipe
+without explicitly defining a subpackage for it. This is an alternative to the
+existing behavior, not an addition to it. See the :ref:`implicit_metapackages`
+section below for more information.
+
+
+Specifying files to include in output
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Files to be included in the package can be specified one of two ways: explicit
+file lists, or scripts that move files into the build prefix.
+
+Explicit file lists are relative paths from the root of the build prefix.
+Explicit file lists support glob expressions. Directory names are also
+supported, and will recursively include contents.
+
+
+.. code-block:: yaml
+
+   outputs:
+     - name: subpackage-name
+       files:
+         - a-file
+         - a-folder
+         - *.some-extension
+         - somefolder/*.some-extension
+
+
+Scripts that create or move files into the build prefix can be any kind of
+script. Known script types need only specify the script name. Currently the list
+of recognized extensions is ``py``, ``bat``, ``ps1``, ``sh``
+
+.. code-block:: yaml
+
+   outputs:
+     - name: subpackage-name
+       script: move-files.py
+
+
+The interpreter command must be specified if the file extension is not
+"recognized."
+
+.. code-block:: yaml
+
+   outputs:
+     - name: subpackage-name
+       script: some-script.extension
+       script_interpreter: program plus arguments to run script
```

Comment:
@msarahan it may be my RPM bias talking but I am not sure if using an external script to organize the sub-packages is a good idea. Anyway, the option to list the files also exists, so I guess it is up to the user decide what s/he wants.